tests/unit/cluster/slot-migration.tcl

```diff
@@ -355,3 +355,23 @@ start_cluster 3 3 {tags {external:skip cluster} overrides {crash-memcheck-enable
         assert_equal {NOREPLICAS Not enough good replicas to write.} $e
     }
 }
+
+start_cluster 3 3 {tags {external:skip cluster regression} overrides {cluster-allow-replica-migration no cluster-node-timeout 1000} } {
```

Comment:
```suggestion
start_cluster 2 0 {tags {external:skip cluster regression} overrides {cluster-allow-replica-migration no cluster-node-timeout 1000} } {
```

We don't need so many nodes, this will make it faster.